sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3336,6 +3336,20 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession {
       checkAnswer(df5, Array.empty[Row])
     }
   }
+
+  test("SPARK-28670: create function should throw AnalysisException if UDF class not found") {
```

Comment:
The test case coverage is not good. 

Could we revert this PR?

Reply:
The revert looks fine to me, but we don't need any feedback to users when resources not found in create functions via `logInfo` or something?